detox/e2e/support/server_api/post.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import client from './client';
```

Comment:
Can we migrate this files to typescript? Not a blocker but it would be preferable

Reply:
I gave it a try but I'm getting lots of error. Honestly, I can't figure out what I'm doing wrong. I created a ticket [MM-28650](https://mattermost.atlassian.net/browse/MM-28650) to address in the future. Will discuss in automation meeting but I will appreciate if anyone from the dev wants to take a stab and make it happen.